src/galaxy/CustomSystem.cpp

```diff
@@ -314,6 +318,49 @@ static luaL_Reg LuaCustomSystemBody_meta[] = {
 	{ 0, 0 }
 };
 
+void CustomSystemBody::LoadFromJson(const Json &obj)
```

Comment:
Are these so different from SystemBody that we can't have a common base class implement this?

Reply:
I was trying to do that when I first wrote this code - at that time the combination of loading the system hierarchy from Lua and SystemBody being unable to be copy-constructed made it infeasible, but I should now be able to implement a common copyable ancestor holding the variables.

Reply:
Implemented a common "data container" used by both CustomSystem and SystemBody.